app/models/user.rb

```diff
@@ -108,4 +104,18 @@ def guest?
   def name
     actor.display_name
   end
+
+  private
+
+  def self.directory_orcid(uid)
+    begin
+      linked_orcid = PsuIdentity::DirectoryService::Client.new.userid(uid).orc_id
+    rescue StandardError => e
+      logger.error(e)
+    end
+
+    OrcidId.new(linked_orcid).to_s
```

Comment:
This line should go inside the `begin...` stanza, otherwise you are trying to parse a nil value, which might work but shouldn't happen

Reply:
_doh_